src/util/apolloPreFetch.js

```diff
@@ -41,7 +41,7 @@ export function handleApolloErrors(handlers = {}, errors, args) {
 		// Call the error handler with the errors and any additional args passed in from the top function
 		const handlerPromise = handler({ graphQLErrors, ...args });
 		// Throw error for malformed error handler functions
-		if (!(handlerPromise instanceof Promise)) {
+		if (typeof handlerPromise.then !== 'function') {
```

Comment:
It turns out that `Promise` is different in each context, so instanceof will not always return true. Testing that `.then` is a function is also a valid way of identifying promises, so I switched to using that.